test/jb_projects/transfer_handle_of.test.js

```diff
@@ -0,0 +1,173 @@
+        ethers.utils.formatBytes32String(PROJECT_HANDLE_2),
+      );
+
+    let storedHandle = await jbProjectsStore.connect(deployer).handleOf(PROJECT_ID_1);
+    let storedProjectId = await jbProjectsStore
+      .connect(deployer)
+      .idFor(ethers.utils.formatBytes32String(PROJECT_HANDLE_2));
+    let storedOldProjectId = await jbProjectsStore
+      .connect(deployer)
+      .idFor(ethers.utils.formatBytes32String(PROJECT_HANDLE_1));
+
+    await expect(storedHandle).to.equal(ethers.utils.formatBytes32String(PROJECT_HANDLE_2));
+    await expect(storedProjectId).to.equal(PROJECT_ID_1);
+    await expect(storedOldProjectId).to.equal(0);
+
+    await expect(tx)
+      .to.emit(jbProjectsStore, 'TransferHandle')
+      .withArgs(
+        PROJECT_ID_1,
+        deployer.address,
+        ethers.utils.formatBytes32String(PROJECT_HANDLE_1),
+        ethers.utils.formatBytes32String(PROJECT_HANDLE_2),
+        projectOwner.address,
+      );
+  });
+
+  it(`Can't transfer handle if is empty handle`, async function () {
+    const { projectOwner, deployer, jbProjectsStore } = await setup();
+
+    await jbProjectsStore
+      .connect(deployer)
+      .createFor(
+        projectOwner.address,
+        ethers.utils.formatBytes32String(PROJECT_HANDLE_1),
+        METADATA_CID,
+      );
+
+    await expect(
+      jbProjectsStore
+        .connect(projectOwner)
+        .transferHandleOf(
+          PROJECT_ID_1,
+          deployer.address,
+          ethers.utils.formatBytes32String(PROJECT_HANDLE_EMPTY),
+        ),
+    ).to.be.revertedWith('0x0a: EMPTY_HANDLE');
+  });
+
+  it(`Can't transfer handle if handle taken already`, async function () {
+    const { projectOwner, deployer, jbProjectsStore } = await setup();
+
+    await jbProjectsStore
+      .connect(deployer)
+      .createFor(
+        projectOwner.address,
+        ethers.utils.formatBytes32String(PROJECT_HANDLE_1),
+        METADATA_CID,
+      );
+
+    await expect(
+      jbProjectsStore
+        .connect(projectOwner)
+        .transferHandleOf(
+          PROJECT_ID_1,
+          deployer.address,
+          ethers.utils.formatBytes32String(PROJECT_HANDLE_1),
+        ),
+    ).to.be.revertedWith('0x0b: HANDLE_TAKEN');
+  });
+
+  it(`Can't transfer handle if not owner of project`, async function () {
+    const { projectOwner, deployer, addrs, jbProjectsStore, mockJbOperatorStore } = await setup();
+
+    await jbProjectsStore
+      .connect(deployer)
+      .createFor(
+        projectOwner.address,
+        ethers.utils.formatBytes32String(PROJECT_HANDLE_1),
+        METADATA_CID,
+      );
+
+    await expect(
+      jbProjectsStore
+        .connect(addrs[0])
+        .transferHandleOf(
+          PROJECT_ID_1,
+          deployer.address,
+          ethers.utils.formatBytes32String(PROJECT_HANDLE_1),
+        ),
+    ).to.be.reverted;
+  });
+
+  it(`Can't transfer handle if operator even with permissions`, async function () {
```

Comment:
this one also doesnt make sense to me

Reply:
You are right, the test need to be opposite (without permissions). Fixed.